content/_help/verify-your-identity/how-to-verify-your-identity._en.md

```diff
@@ -60,8 +60,8 @@ Login.gov is a secure, government website that adheres to the highest standards
     * Click “Continue”
 1.  Enter your social security number and click “Continue”
 1.  Verify the information you have submitted so far is correct and click “Continue”
-1.  Enter a phone number with your name on the plan. This is the last step for identity verification.
-    * If you do not have a phone plan with your name on it or a U.S. phone number, click on “Get a code via mail instead.” You will have to wait to receive a letter in the mail and then follow the instructions to enter the code.
+1.  Enter your primary phone number. This is the last step for identity verification.
+    * If you do not have a U.S. phone number or cannot verify by phone, click on “Get a code via mail instead.” You will have to wait to receive a letter in the mail and then follow the instructions to enter the code.
```

Comment:
Initially I was thinking about it in terms of keeping translations in sync, but it occurs to me that "Get a code via mail instead" is not the text we use in the IdP, so this help guide could be confusing for someone looking for that exact label.

From what I can tell, the label is ["Verify your address by mail instead"](https://github.com/18F/identity-idp/blob/4b6f8b7ecd8b1515acd297f8ce15e9a6cbff61c7/config/locales/idv/en.yml#L204) (["Verifique su dirección por correo"](https://github.com/18F/identity-idp/blob/4b6f8b7ecd8b1515acd297f8ce15e9a6cbff61c7/config/locales/idv/es.yml#L213) in Spanish, ["Vérifiez plutôt votre adresse par courrier"](https://github.com/18F/identity-idp/blob/4b6f8b7ecd8b1515acd297f8ce15e9a6cbff61c7/config/locales/idv/fr.yml#L225) in French).

I think it's not directly related to the changes you're trying to make here, so would be fine to update separately to avoid blocking this if you'd prefer.

Reply:
Makes sense to keep this consistent. I'll make the update!